### DISCUSSION TITLE
Renovate PRs from Dependabot Alerts

### DISCUSSION THREAD

**Discussion by gavintoddchurch** [renovatebot/renovate#26328] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I am trying to understand what the expected behavior for Renovate is on Dependabot alerts. I have a repo that has a dependabot critical alert on babel that is in the package-lock.json but not the package.json. In this scenario is Renovate expected to raise a PR from the dependabot alert or what does Renovate "read" from Dependabot. https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Not all alerts are possible (theoretically) to be remediated, and not all which are theoretically possible can be remediate in practicality, due to Renovate limitations. Most likely you're seeing the latter case. If you can create a public reproduction repo then we can dig into it in more detail to work out if it can be supported in future.

**Reply by gavintoddchurch** [reply]
Thanks for getting back to me. I'm not too worried about the update so I don't think i'll create the reproduction I was just trying to get a better understand of what happens where dependabot has a critical alert and what is expected from Renovate.

In general if there is a dependabot alert Renovate will try to resolve it if it can though? Is that when you get the [SECURITY] label added to the Renovate PR?

**Reply by gavintoddchurch** [reply]
@rarkins When you get some time could take a look and see why Renovate couldn't resolve this dependabot alert? 
https://github.com/GavinToddlds/renovate-dependabot-test

Thanks for all your help

**Reply by gavintoddchurch** [reply]
Any ideas why the dependabot critical security alert isn't causing renovate to fix it?
